br/pkg/restore/client.go

```diff
@@ -2017,6 +2174,22 @@ func (rc *Client) SaveSchemas(
 	return nil
 }
 
+// InitFullClusterRestore init fullClusterRestore and set SkipGrantTable as needed
+func (rc *Client) InitFullClusterRestore(explicitFilter bool) {
+	rc.fullClusterRestore = !explicitFilter && rc.IsFull()
```

Comment:
if user set `-f "test.*" and do not want to restore system tables. shall we still check system table is compatible and set `config.GetGlobalConfig().Security.SkipGrantTable = true`?

Reply:
with explicit filter, fullClusterRestore=false, so no check and won't set SkipGrantTable

Reply:
if user use `br backup full` to generate backupMeta file.
and then use `br restore -f "test.*"` to restore some of tables from full backup.

In above situation. I think it will checkSystem compatible and set SkipGrantTable and which is necessary.

Reply:
OK，I see.